ngraph/core/include/openvino/pass/serialize.hpp

```diff
@@ -74,10 +76,14 @@ class OPENVINO_API StreamSerialize : public ov::pass::FunctionPass {
 
     bool run_on_function(std::shared_ptr<ov::Function> f) override;
 
+    OPENVINO_DEPRECATED("This constructor is deprecated. Please use new extension API")
     StreamSerialize(std::ostream& stream,
                     std::map<std::string, ngraph::OpSet>&& custom_opsets = {},
                     const std::function<void(std::ostream&)>& custom_data_serializer = {},
                     Serialize::Version version = Serialize::Version::UNSPECIFIED);
```

Comment:
let's remove these ctors. This API was not targeted for end users in 2021.4 and earlier

Reply:
We cannot remove this ctor, because it is used in the CPU plugin, which can serialize networks with old extensions

Reply:
can CPU plugin use new extension mechanism?

Reply:
CPU plugin still can have some operations which were created by old extensibility mechanism. In this case we need to have information about these opetations.